src/test/scala/dxWDL/base/ExtrasTest.scala

```diff
@@ -5,19 +5,30 @@ import com.dnanexus.exceptions.ResourceNotFoundException
 import dxWDL.compiler.EdgeTest
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
-
 import spray.json._
 import wdlTools.eval.WdlValues
 import DefaultJsonProtocol._
+import wdlTools.util.Util
 
 class ExtrasTest extends AnyFlatSpec with Matchers {
   val verbose: Verbose = Verbose(on = true, quiet = true, Set.empty)
   val verbose2: Verbose = Verbose(on = false, quiet = true, Set.empty)
 
   private def getIdFromName(name: String): String = {
```

Comment:
Wouldnt calling this function with "app-cloud_workstation" instead of "cloud_workstation" fix this issue?

Seems that we could always just use precise IDs

Reply:
yes you are correct